File: web/src/api/normal/frp_client_cfg.js
import request from "../../request/index.js";

const baseUrl = '/v1/normal/frp/client/cfg'


export const apiNormalFrpClientCfgPage = (data) => {
    return request.request({
        url: `${baseUrl}/page`,
        method: 'POST',
        data
    })
}

export const apiNormalFrpClientCfgCreate = (data) => {
    return request.request({
        url: `${baseUrl}/create`,
        method: 'POST',
        data
    })
}

export const apiNormalFrpClientCfgMigrate = (data) => {
    return request.request({
        url: `${baseUrl}/migrate`,
        method: 'POST',
        data
    })
}

export const apiNormalFrpClientCfgEnable = (data) => {
    return request.request({
        url: `${baseUrl}/enable`,
        method: 'POST',
        data
    })
}

export const apiNormalFrpClientCfgUpdate = (data) => {
    return request.request({
        url: `${baseUrl}/update`,
        method: 'POST',
        data
    })
}


export const apiNormalFrpClientCfgDelete = (data) => {
    return request.request({
        url: `${baseUrl}/delete`,
        method: 'POST',
        data
    })
}

File: server/router/normal/frp_client_cfg.go
package normal

import (
	"github.com/gin-gonic/gin"
	"server/controller/normal/frp_client_cfg"
	"server/global"
	"server/router/middleware"
)

func InitFrpClientCfg(group *gin.RouterGroup) {
	g := group.Group("frp/client/cfg", middleware.Auth(global.Jwt))
	g.POST("create", frp_client_cfg.Create)
	g.POST("migrate", frp_client_cfg.Migrate)
	g.POST("update", frp_client_cfg.Update)
	g.POST("enable", frp_client_cfg.Enable)
	g.POST("delete", frp_client_cfg.Delete)
	g.POST("page", frp_client_cfg.Page)
}
